libs/providers/flipt-web/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@openfeature/flipt-web-provider",
+  "version": "0.0.1",
```

Comment:
I recommend you match this with the version in the .release-please-manifest.json.

I think release please will take care of it anyway, but let's avoid the confusion.